src/app/providers/react-query.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { Suspense } from "react"
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query"
+
+export const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+    },
+  },
+})
+
+interface ReactQueryProps {
+  children: React.ReactNode
+}
+
+const ReactQuery = (props: ReactQueryProps) => {
+  return (
+    <Suspense>
+      <QueryClientProvider client={queryClient}>{props.children}</QueryClientProvider>
+    </Suspense>
+  )
+}
+
+export default ReactQuery
```

Comment:
Don't forget to create a index.ts file and reexport all providers in order to import them from one place

Reply:
Done